pandas/compat/numpy/__init__.py

```diff
@@ -21,6 +21,21 @@
     )
 
 
+np_long: type
+np_ulong: type
+
+if _nlv >= Version("2.0.0.dev0"):
+    try:
+        np_long = np.long  # type: ignore[attr-defined]
+        np_ulong = np.ulong  # type: ignore[attr-defined]
+    except AttributeError:
+        np_long = np.int_
+        np_ulong = np.uint
+else:
+    np_long = np.int_
```

Comment:
So just for my understanding, `np.long` was newly added to the top level namespace/didn't exist before a certain version? It would be preferable to already use `np.long/np.ulong` if possible